CODE_OF_CONDUCT.md

```diff
@@ -1,5 +1,25 @@
 # Contributor Covenant Code of Conduct
 
+* **Last Updated:** 02/07/2021
```

Comment:
Not really necessary? Git shows commits

Reply:
A bit automatic, though understandable. Deleting rn.